src/codegen/program.lisp

```diff
@@ -40,6 +40,14 @@
           :else :if (and (typep thing '(cons (member cl:defstruct)))
                          (not (sub-struct-p thing)))
                   :collect thing :into defstructs
+          ;; (DEFCLASS (FOO ... (:INCLUDE ...
+          :if (and (typep thing '(cons (member cl:defclass)))
```

Comment:
Two issues:

1. The comment should read `(DEFCLASS FOO (...) ...)`
2. You need an `:else` in front of this `:if`!

The second point is causing things to be defined multiple times.